src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -192,6 +193,11 @@ function validate_options_type_by_value(input_options_with_data) {
             const details = `type of flag ${option} should be ${type_of_option} (and the received value is ${value})`;
             throw_cli_error(ManageCLIError.InvalidArgumentType, details);
         }
+        // special case for unset value (specified by '').
+        if (!LIST_UNSETTABLE_OPTIONS.includes(option) && value === '') {
```

Comment:
This condition would also works/validates the leading space check for the flags and we can put the error msg accordingly